microproxy/protocol/http2.py

```diff
@@ -38,6 +39,8 @@ def decorate(*args):
 
         self.stream = stream
 
+        self.on_pending_events = on_pending_events or on_unhandled_with_type("PENDING_EVENTS")
+        self.on_header_receive = on_header_receive or on_unhandled_with_type("REQUEST_HEADER_RECEIVE")
```

Comment:
If you have this event, then I think we SHOULD NOT have request event and response, but with header event and data event.

Because the header event will be sent twice with on_header_receive callback and on_request or on_response callback...